src/event/server_events.c

```diff
@@ -1199,6 +1199,10 @@ sv_ev_muc_occupant_online(const char *const room, const char *const nick, const
 
     gboolean updated = muc_roster_add(room, nick, jid, role, affiliation, show, status);
```

Comment:
I think sometimes we dont get a jid here jsut the nick of the user. So later on jid will be NULL and strdup() will segfault.

Reply:
The API of `muc_members_*` expect a jid. I pushed a fix that ensure that the API is correctly called.